benches/oneshot/ipa.rs

```diff
@@ -16,10 +17,12 @@ async fn main() -> Result<(), Error> {
     const MAX_QUERY_SIZE: usize = 1000;
     const NUM_USERS: usize = 300;
     const MAX_RECORDS_PER_USER: usize = 10;
+    type BenchField = Fp32BitPrime;
 
     let mut config = TestWorldConfig::default();
-    config.gateway_config =
-        GatewayConfig::symmetric_buffers((NUM_USERS * MAX_RECORDS_PER_USER).clamp(16, 1024));
+    config.gateway_config = GatewayConfig::symmetric_buffers::<BenchField>(
+        (NUM_USERS * MAX_RECORDS_PER_USER).clamp(16, 1024),
```

Comment:
@richajaindce made a change elsewhere to remove the variability in this test.  Look at #561.

What happened to #459, by the way?  I kinda liked that, but I guess got too ambitious.

Reply:
yea I think #459 tried to bite more than I could swallow. We probably need something simpler that just adds parameters to this IPA benchmark. Naturally we could see where it is going and course-correct if necessary